src/sqlancer/postgres/PostgresProvider.java

```diff
@@ -117,11 +117,21 @@ public enum Action implements AbstractAction<PostgresGlobalState> {
         }), //
         RESET_ROLE((g) -> new SQLQueryAdapter("RESET ROLE")), //
         COMMENT_ON(PostgresCommentGenerator::generate), //
-        RESET((g) -> new SQLQueryAdapter("RESET ALL") /*
-                                                       * https://www.postgresql.org/docs/devel/sql-reset.html TODO: also
-                                                       * configuration parameter
-                                                       */), //
-        NOTIFY(PostgresNotifyGenerator::createNotify), //
+        RESET((g) -> {
+            StringBuilder sb = new StringBuilder();
+            sb.append("RESET ");
+
+            // List of configuration parameters
+            String[] configParameters = { "statement_timeout", "timezone", "log_statement", "random_page_cost",
```

Comment:
This should be a static variable. It would be better to create a separate class for this generator.